server/src/routes/api/v1/podcastVotesRouter.js

```diff
@@ -0,0 +1,105 @@
+import express from 'express'
+import objection from 'objection'
+const { ValidationError } = objection
+
+import { Vote } from '../../../models/index.js'
+
+const podcastVotesRouter = new express.Router({ mergeParams: true })
+
+podcastVotesRouter.get('/', async (req, res) => {
+  const { podcastId } = req.params
+  let userId
+  if (req.user) userId = req.user.id
+
+  try {
+    let votes = {}
+    votes.userVotes = await Vote.query()
+      .where('podcastId', podcastId)
+      .andWhere('userId', userId)
+      .first()
+    votes.totalVotes = await Vote.query()
+      .where('podcastId', podcastId)
+      .sum('value as value')
+      .first()
+    return res.status(201).json({ votes: votes })
+  } catch (error) {
+    if (error instanceof ValidationError) {
+      return res.status(422).json({ errors: error.data })
+    }
+    return res.status(500).json({ errors: error })
+  }
+})
+
+podcastVotesRouter.post('/', async (req, res) => {
+  const { value } = req.body
+  const { podcastId } = req.params
+  const userId = req.user.id
+
+  try {
+    let newVote = {}
+    newVote.userVotes = await Vote.query()
+      .insert({ podcastId, userId, value })
+      .first()
+      .returning('*')
+    newVote.totalVotes = await Vote.query()
+      .where('podcastId', podcastId)
+      .sum('value as value')
+      .first()
+    return res.status(201).json({ votes: newVote })
```

Comment:
you should not be assigning `totalVotes` as a property of `newVote`. totalVotes is relevant to a Podcast, note a vote (a vote shouldnt know the amount of total vote for a specific podcast). 
Instead, in your `.json`, return two poperties, `votes` and `totalVotes`